lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -372,8 +372,9 @@ export class PublicClientApplication {
             return await this.authModule.getValidToken(silentRequest);
         } catch (e) {
             const isServerError = e instanceof ServerError;
+            const isInteractionRequiredError = e instanceof InteractionRequiredAuthError;
```

Comment:
I think we should be consistent in this logic, given there is a static method on `InteractionRequiredAuthError` that also answers this question.

Reply:
The static method determines whether or not we should create an `InteractionRequiredAuthError`. I think this answers the question of whether or not it actually is one. Similar but slight difference and I would personally err towards checking what it is rather than what it should be.

Reply:
Ah, good point.